engine/src/main/java/org/terasology/logic/characters/CharacterMovementComponent.java

```diff
@@ -28,6 +28,9 @@
 import java.util.List;
 
 /**
+ * This component is attached to all character entities. It governs movement and stores
+ * associated paramenters. <br/>
+ * The AliveComponent should necessarily be attached to the character entity for the movement systems to work.
```

Comment:
`AliveComponent` vs `CharacterAliveComponent` - was renamed but remains old name in doc :-)